lib/modules/datasource/maven/index.ts

```diff
@@ -249,59 +249,66 @@ export class MavenDatasource extends Datasource {
     }
 
     const cacheNs = 'datasource-maven:head-requests';
+    const cacheTimeoutNs = 'datasource-maven:head-requests-timeout';
```

Comment:
It stores cache valid flags separately.
Otherwise, there is a risk of cache never being invalidated since we update it too often.